groups/sig-etcd/groups.yaml

```diff
@@ -23,10 +23,21 @@ groups:
       - jablair@redhat.com
       - siarkowicz@google.com
       - wenjiazhang@google.com
+      - ivan@vald.es
     settings:
       AllowWebPosting: "true"
       ReconcileMembers: "true"
       WhoCanPostMessage: "ANYONE_CAN_POST"
       WhoCanViewGroup: "ALL_MEMBERS_CAN_VIEW"
       WhoCanModerateContent: "OWNERS_AND_MANAGERS"
       MessageModerationLevel: "MODERATE_NONE"
+
+  - email-id: k8s-infra-staging-etcd@kubernetes.io
+    name: k8s-infra-staging-etcd
+    description: |-
+      ACL for staging etcd
+    settings:
+      ReconcileMembers: "true"
+    members:
+    - sig-etcd-leads
```

Comment:
> 2025/07/29 23:11:43 unable to add sig-etcd-leads to "k8s-infra-staging-etcd@kubernetes.io" as MEMBER: googleapi: Error 400: Invalid Input: memberKey, invalid

https://prow.k8s.io/view/gs/kubernetes-ci-logs/logs/post-k8sio-groups/1950333262106726400

Reply:
Good to know. Thanks, Ben.